apps/api/src/app/workflows/workflow.controller.ts

```diff
@@ -202,4 +207,32 @@ export class WorkflowController {
       })
     );
   }
+
+  @Post('/overrides')
+  @UseGuards(RootEnvironmentGuard)
+  @ApiResponse(CreateWorkflowOverrideResponseDto)
+  @ApiOperation({
+    summary: 'Create workflow override',
+    description:
+      'In order to create workflow override please make sure to identify the workflow by triggerIdentifier or _workflowId, and tenant by tenantIdentifier',
+  })
+  @ExternalApiAccessible()
+  createWorkflowOverride(
+    @UserSession() user: IJwtPayload,
+    @Body() body: CreateWorkflowOverrideRequestDto,
+    @Param('workflowId') workflowId: string
```

Comment:
redundant 🧹
```suggestion
```

Reply:
removed in the next branch 'NV-3135-add-update-workflow-override-use-case'

Reply:
I actually feel like we should have this endpoint be like `POST /workflows/:workflowId/overrides` and then we won't need to have the `_workflowId` nor `triggerIdentifier` in the request body. We only use the `triggerIdentifier` to trigger the workflow, in all other cases like get/update/delete/status we use `workflowId`.

Reply:
Actually, it makes a lot of sense to me, i make it like this because of our sync before the cycle. i thought we wanted to couple it to the workflow entity this way.

Do you mind if we merge all of the existing branches to `workflow-override` so i could then create a new PR in order to refactor the location of all of the routes altogether?